tools/generator/templates/player/rust/Makefile-rust.jinja2

```diff
@@ -0,0 +1,23 @@
+{# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2020 Association Prologin <association@prologin.org>
+#}
+{% extends "includes/Makefile-language.jinja2" %}
+
+{% block lang %}rust{% endblock %}
+
+{% block extra_flags %}
+RUSTC_FLAGS = --edition 2018 -g -C relocation-model=pic
+CPPFLAGS += -MMD -MP
+LDFLAGS = -lm -lrt -ldl -lpthread -lstdc++
+{% endblock %}
+
+{% block extra_rules %}
+objs += $(CHAMPION_TARGET).o
```

Comment:
Please hardcode "champion" for consistency, or change it in all the other languages